MetaTempUnit/Hardware/library/KelvinVR/src/KelvinVR.cpp

```diff
@@ -267,6 +270,57 @@ void Kelvin::printHelp()
   Serial.println("  LED FADE RED - Fade red LED");
 }
 
+void Kelvin::ledAction(const String &command)
```

Comment:
[nit] Testing required for this commands, please test it in the lab before merging